pkg/endpoint/regenerationcontext.go

```diff
@@ -46,6 +47,10 @@ type regenerationContext struct {
 }
 
 func ParseExternalRegenerationMetadata(ctx context.Context, c context.CancelFunc, e *regeneration.ExternalRegenerationMetadata) *regenerationContext {
+	if e.RegenerationLevel == regeneration.Invalid {
+		log.WithField(logfields.Reason, e.Reason).Errorf("Uninitialized regeneration level")
```

Comment:
Please add this to the CI log failures as well so we don't ignore it.

Reply:
Done!